packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx

```diff
@@ -133,32 +134,35 @@ const InputContainer = withBaseElementProps(View, {
 
 export const FieldControl: FieldControl = () => {
   const { input, setInput } = React.useContext(InputContext);
+  const handleSendMessage = React.useContext(SendMessageContext);
 
   const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     (e.target as HTMLFormElement).reset();
 
+    const submittedContent: InputContent[] = [];
     if (input?.text) {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars -- TODO send message
-      const textContent: TextContent = {
-        type: 'text',
-        value: input.text,
+      const textContent: InputContent = {
+        text: input.text,
       };
+      submittedContent.push(textContent);
     }
     if (input?.files) {
-      input.files.map((file) => {
+      input.files.forEach((file) => {
         file.arrayBuffer().then((buffer) => {
           // eslint-disable-next-line @typescript-eslint/no-unused-vars -- TODO send message
```

Comment:
I think this comment can be removed now?

Reply:
yep! removed